### PR TITLE
Fabric 1.19 support

### PR DESCRIPTION
**Pull request by Aurelien30000**
Obviously tested on a basic server.

### CODE REVIEW

File: fabric/src/main/java/me/lucko/luckperms/fabric/FabricCommandExecutor.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public FabricCommandExecutor(LPFabricPlugin plugin) {
     }
 
     public void register() {
-        CommandRegistrationCallback.EVENT.register((dispatcher, dedicated) -> {
+        CommandRegistrationCallback.EVENT.register((dispatcher, dedicated, environment) -> {
```
Comment (haykam821) [inline fabric/src/main/java/me/lucko/luckperms/fabric/FabricCommandExecutor.java:56]:
```suggestion
        CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> {
```

Comment (Aurelien30000) [reply fabric/src/main/java/me/lucko/luckperms/fabric/FabricCommandExecutor.java:56]: it makes more sense indeed, ty!

**Comment by lucko** [comment]
Awesomeeeee! thank you!!

**Comment by haykam821** [comment]
It looks like [the build](https://ci.lucko.me/job/LuckPerms/1435/console) ran out of memory.

**Pull request merged**